lib/rocket_pants/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Railtie
 
     initializer "rocket_pants.setup_caching" do |app|
       if RocketPants.caching_enabled?
-        app.middleware.insert 'Rack::Runtime', RocketPants::CacheMiddleware
```

Comment:
We need to maintain compatibility with rails 4, can you add a check for determining the rails version and use the old way for that?